src/hooks/useOutsideAlerter.tsx

```diff
@@ -21,5 +21,9 @@ export default function OutsideAlerter({ children, onClose }: any) {
 
     useOutsideAlerter(wrapperRef, onClose);
 
-    return <div ref={wrapperRef}>{children}</div>;
+    return (
+        <div ref={wrapperRef} style={{ display: 'inline-block', width: 'auto' }}>
+            {children}
+        </div>
+    );
```

Comment:
I'm not happy with an hook returning an element.

Why can't we have a hook like this:
https://github.com/impactMarket/web/blob/master/src/hooks/useClickOutside.ts

Then apply to the composed element, passing the ref?